src/main/java/org/hl7/davinci/priorauth/App.java

```diff
@@ -108,6 +123,9 @@ public static ModelResolver getModelResolver() {
    * @return true if debugMode is true, false otherwise
    */
   public static boolean isDebugModeEnabled() {
-    return debugMode;
+    System.out.println("4-1debug mode in isDebugEnabled is: " + debugMode);
+    System.out.println("4-2debug mode in isDebugEnabled is: " + config.getProperty("debugMode"));
+    return true;
+    // return debugMode;
```

Comment:
hard coding debug mode to always true as temporary work around - debugMode variable changes to false at end of Gradle bootRun with springboot version upgrade. Not sure why this behavior occurs